pandas/tests/test_categorical.py

```diff
@@ -4601,6 +4601,45 @@ def f():
         df = pd.DataFrame({'a': pd.Categorical(idx)})
         tm.assert_frame_equal(df.fillna(value=pd.NaT), df)
 
+    @pytest.mark.parametrize('fill_value, expected_output', [
+        ('a', ['a', 'a', 'b', 'a', 'a']),
+        ({1: 'a', 3: 'b', 4: 'b'}, ['a', 'a', 'b', 'b', 'b']),
+        ({1: 'a'}, ['a', 'a', 'b', np.nan, np.nan]),
+        ({1: 'a', 3: 'b'}, ['a', 'a', 'b', 'b', np.nan]),
+        (pd.Series('a'), ['a', np.nan, 'b', np.nan, np.nan]),
+        (pd.Series('a', index=[1]), ['a', 'a', 'b', np.nan, np.nan]),
+        (pd.Series({1: 'a', 3: 'b'}), ['a', 'a', 'b', 'b', np.nan]),
+        (pd.Series(['a', 'b'], index=[3, 4]), ['a', np.nan, 'b', 'a', 'b'])
+    ])
+    def test_fillna_series_categorical(self, fill_value, expected_output):
+        # GH 17033
+        # Test fillna for a Categorical series
+        data = ['a', np.nan, 'b', np.nan, np.nan]
+        s = pd.Series(pd.Categorical(data, categories=['a', 'b']))
```

Comment:
`Series` and `Categorical` have already been imported so you can remove the `pd.` throughout the tests you added.  Instances of this in existing tests should already be covered by #18277.

Reply:
Thanks, I'll change that.